tools/validators/instance_validator/tests/instance_parser_test.py

```diff
@@ -48,5 +48,22 @@ def testInstanceValidatorParseProperFormat(self):
         _TESTCASE_PATH, 'GOOD', 'good_building_type.yaml'))
     self.assertTrue(parsed_yaml.data)
 
+  def testInstanceValidatorParseProperConnections(self):
+    parsed_yaml = instance_parser.parse_yaml(os.path.join(
+        _TESTCASE_PATH, 'GOOD', 'good_building_connections.yaml'))
+    self.assertTrue(parsed_yaml.data)
```

Comment:
check contents?

Reply:
For this testfile, I'm focusing only on whether the syntax parsing detects YAML files that follow the YAML schema. These tests make sure the first layer of validation works; the `entity_instance_test` is where I make sure the content-checking layer of validation works.